firmware/hw_layer/sensors/CJ125.cpp

```diff
@@ -290,8 +290,19 @@ static void cjStart(void) {
 		return;
 	}
 	
+	// Software reset cj125
+	for (int i = 0; i < 2; i++)
```

Comment:
can you please add code comment clarifying this voodoo?

Reply:
I don't fully understand this voodoo either

Reply:
I need to go actually diagnose whatever my weird SPI problem is, which will explain this voodoo (or obviate its existence)

Reply:
why did you add it? was it helping with something? can you add a comment on what is seems to affect?

Reply:
The problem I was trying to get around here is that the SPI controller seemed to be sending a garbage first byte, which confused the cj125.  The first reset comes out as garbage, and the second resets it and puts it in a predictable (ie normal) state.  I should sort that out more completely before this is merged.